src/html/partials/concept.html

```diff
@@ -1,4 +1,73 @@
 <section class="concept-section">
-  <h2 class="sm-title">The Concept</h2>
-  <p class="bg-intro">Meet Florence</p>
+
+  <div class="concept-section__intro">
+    <h2 class="sm-title">The Concept</h2>
+    <p class="bg-intro">Meet Florence</p>
```

Comment:
I keep thinking "background" when I see "bg" here (although I know it's supposed to mean "big"). Might want to be more explicit with the naming, e.g. "big-intro" or just "big-text" that way it could potentially be used for other non-intro pieces of text if needed.

Reply:
what about `lg` for large?

Reply:
Will do, I think those both make a lot more sense.